### ISSUE TITLE
Can not compile 0.8.0

### ISSUE THREAD

**Issue by parazyd** [maciejhirsz/tiny-bip39#30]
Hi, I'm getting the following error when trying to compile v0.8.0.
I've made a new Cargo.toml and have only this for the deps:

```
[dependencies]
tiny-bip39 = "0.8.0"
```

```
$ cargo build
   Compiling tiny-bip39 v0.8.0
error[E0184]: the trait `Copy` may not be implemented for this type; the type has a destructor
   --> /home/parazyd/.cargo/registry/src/github.com-1ecc6299db9ec823/tiny-bip39-0.8.0/src/language.rs:116:24
    |
116 | #[derive(Debug, Clone, Copy, PartialEq, Zeroize)]
    |                        ^^^^ Copy not allowed on types with destructors
    |
    = note: this error originates in the derive macro `Copy` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0184`.
error: could not compile `tiny-bip39` due to previous error
```

Rust/Cargo version is `1.55.0-nightly`.

**Comment by zastrozzi** [comment]
Same issue here

**Comment by maciejhirsz** [comment]
Thanks for the report, released 0.8.1 with a fix.
